extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -60,7 +60,7 @@ define('ext.wikia.adEngine.template.porvata', [
 	}
 
 	function isVpaid(contentType) {
-		return contentType.indexOf('application/') === 0;
+		return contentType === 'application/javascript';
```

Comment:
`application/x-mpegurl` is not a vpaid 🙈 
I checked the data in Qubole from whole month and there is only `application/javascript` vpaid type